arm-recoveryservicesbackup/2016-06-01/swagger/recoveryservicesbackup.json

```diff
@@ -0,0 +1,1548 @@
+          "type": "string"
+        },
+        "startTime": {
+          "format": "date-time",
+          "description": "Operation start time.",
+          "type": "string"
+        },
+        "endTime": {
+          "format": "date-time",
+          "description": "Operation end time.",
+          "type": "string"
+        },
+        "error": {
+          "$ref": "#/definitions/OperationStatusError",
+          "description": "Error information related to this operation."
+        },
+        "properties": {
+          "$ref": "#/definitions/OperationStatusExtendedInfo",
+          "description": "Additional information associated with this operation."
+        }
+      }
+    },
+    "OperationStatusError": {
+      "description": "Error information associated with operation status call.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "Error code of the operation failure.",
+          "type": "string"
+        },
+        "message": {
+          "description": "Error message displayed if the operation failure.",
+          "type": "string"
+        }
+      }
+    },
+    "OperationStatusExtendedInfo": {
+      "description": "Base class for additional information of operation status.",
+      "type": "object",
+      "properties": {}
+    },
+    "ResourceList[Job]": {
+      "description": "Resource list wrapper for \"T\" type",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Gets or sets the list of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Resource[Job]"
+          }
+        },
+        "nextLink": {
+          "description": "Gets or sets the uri to fetch the next page of resources. Call ListNext() fetches next page of resources.",
+          "type": "string"
+        }
+      }
+    },
+    "Resource[Job]": {
+      "description": "Resource wrapper for \"T\" type",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Resource Id represents the complete path to the resource",
+          "type": "string"
+        },
+        "name": {
+          "description": "Resource name associated with the resource",
+          "type": "string"
+        },
+        "type": {
+          "description": "Resource type represents the complete path of the form Namespace/ResourceType/ResourceType/...",
+          "type": "string"
+        },
+        "location": {
+          "description": "Resource location",
+          "type": "string"
+        },
+        "tags": {
+          "description": "Resource tags",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "eTag": {
+          "description": "Optional ETag",
+          "type": "string"
+        },
+        "properties": {
+          "$ref": "#/definitions/Job",
```

Comment:
Please apply the `"x-ms-client-flatten": true` extension over here. This makes things easy for the customer as he/she does not have to access `Job.properties.foo` every time.